backend/globaleaks/handlers/wizard.py

```diff
@@ -71,6 +71,10 @@ def wizard(store, request, language):
 
         db_create_admin_user(store, admin_dict, language)
 
+        # Add the new admin's email to list of recipients of errors
+        notif = NotificationFactory(store)
+        notif.get_val('exception_email_address_list').append(admin_dict['mail_address'])
```

Comment:
the reason why this would not work are:
- the admin email can change during time, so that this setting will work at initialization and will require manual management.
- the administrators can be more than
- existing nodes (that are the one from which we have more interest in notifying them the issues) will have to manually configure it.